src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/FormPassTest.php

```diff
@@ -37,47 +38,37 @@ public function testDoNothingIfFormExtensionNotLoaded()
 
     public function testAddTaggedTypes()
     {
-        $container = new ContainerBuilder();
-        $container->addCompilerPass(new FormPass());
-
-        $extDefinition = new Definition('Symfony\Component\Form\Extension\DependencyInjection\DependencyInjectionExtension');
-        $extDefinition->setArguments(array(
-            new Reference('service_container'),
-            array(),
-            array(),
-            array(),
-        ));
+        $container = $this->createContainerBuilder();
 
-        $container->setDefinition('form.extension', $extDefinition);
+        $container->setDefinition('form.extension', $this->createExtensionDefinition());
         $container->register('my.type1', __CLASS__.'_Type1')->addTag('form.type');
         $container->register('my.type2', __CLASS__.'_Type2')->addTag('form.type');
 
         $container->compile();
 
         $extDefinition = $container->getDefinition('form.extension');
-
-        $this->assertEquals(array(
-            __CLASS__.'_Type1' => 'my.type1',
-            __CLASS__.'_Type2' => 'my.type2',
-        ), $extDefinition->getArgument(1));
+        $locator = $extDefinition->getArgument(0);
+
+        $this->assertInstanceOf(ServiceLocatorArgument::class, $locator);
```

Comment:
No need for `assertInstanceOf` as `assertEquals` fails on an type-class mismatch.